cmd/deploy/deploy.go

```diff
@@ -459,10 +459,6 @@ func setDeployOptionsValuesFromManifest(ctx context.Context, deployOptions *Opti
 			deployOptions.Manifest.Deploy.ComposeSection.ComposesInfo[0].ServicesToDeploy = deployOptions.servicesToDeploy
 		}
 	}
-
-	if len(deployOptions.servicesToDeploy) == 0 {
```

Comment:
@AgustinRamiroDiaz Do you know why we were deploying name if services to deploy were empty

Reply:
I think it was deploys that are NOT docker-compose